betterdiscordctl

```diff
@@ -59,13 +63,13 @@ EOF
 verbose() {
   if (( verbosity >= $1 )); then
     shift
-    printf '%s\n' "$1" >&2
+    printf "%s\n" "$1" >&2
```

Comment:
Why double quotes?

Reply:
Mildly nice to `printf`, because it leaves parsing `\n` to the shell parser and not the builtin at runtime. I think it's mildly preferred. Doesn't really matter; it's shell anyways.